www/grpc/grpc-gateway.config.yaml

```
@@ -25,6 +25,9 @@ http:
     - selector: pactus.Blockchain.GetBlockchainInfo
       get: "/v1/blockchain/info"
 
+    - selector: pactus.Blockchain.GetPublicKey
+      get: "/v1/blockchain/public_key/address/{address}"
```

```suggestion
      get: "/v1/blockchain/public_keys/address/{address}"
```

Please consult with @Ja7ad